contracts/weapons.sol

```diff
@@ -416,6 +421,10 @@ contract Weapons is Initializable, ERC721Upgradeable, AccessControlUpgradeable,
             if(wbp.fiveStarBurnPoints > 10)
                 wbp.fiveStarBurnPoints = 10;
         }
+        wbp.lowStarBurnPoints = uint8(carriedLowStarBurnPoints.add(wbp.lowStarBurnPoints));
+        wbp.fourStarBurnPoints = uint8(carriedFourStarBurnPoints.add(wbp.fourStarBurnPoints));
+        wbp.fiveStarBurnPoints = uint8(carriedFiveStarBurnPoints.add(wbp.fiveStarBurnPoints));
+
```

Comment:
Since the code that clamps each value to the cap is all done before this, it is entirely possible for each of these bonuses to exceed the cap. This is probably not what we want.

Reply:
Fixed. I was thinking for a while how to do that the best, either add another ton of ifs for checking if each carried bonus is > 0 and then if new bonus does not exceed the limit or just check for everything by default every time we do a reforge. The later one seems like a better option.